internal/service/federatedsettingsidentityprovider/resource_federated_settings_identity_provider.go

```diff
@@ -230,6 +295,11 @@ func resourceMongoDBAtlasFederatedSettingsIdentityProviderUpdate(ctx context.Con
 		return diag.FromErr(fmt.Errorf("error retreiving federation settings identity provider (%s): %s", federationSettingsID, err))
 	}
 
+	if d.HasChange("protocol") {
```

Comment:
related to a question above, can be only 1 or 2 protocols at the same time?

Reply:
Yes, if you only change the protocol it fails

Reply:
how does it fail? is it a readable error?

Reply:
Unexpected error from the server. From looking at the [code](https://github.com/10gen/mms/blob/6850f30e48c88c6bd5d596ec386276c06e79705c/server/src/main/com/xgen/cloud/federation/_public/svc/IdentityProviderSvcProxy.java#L81), updating protocol or trying to update from SAML to OIDC or the other way around is not possible